app/views/components/pickadate.html

```diff
@@ -4,3 +4,11 @@
   </span>
   <input type="text" placeholder="Select a date..." class="form-control" ng-disabled="disabled">
 </div>
+<div ng-if="minDate || maxDate" class="help-block row">
+   <label class="valid-range-msg" translate>Valid Range:</label>
```

Comment:
Does this need to be in a label? It is styled the same as the rest of the text and you had to add `font-weight:normal;` to make it look the same.